### PR TITLE
mono: relocate special folders

### PR DESCRIPTION
**Pull request by ymartin59**
_Motivation:_ Preserve certificates store from DSM upgrades
_Linked issues:_ #3150 

### Checklist
- [X] Build rule `all-supported` completed successfully (except not supported PPC architectures)
- [ ] Package upgrade completed successfully
- [X] New installation of package completed successfully

### CODE REVIEW

File: spk/mono/src/service-setup.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+}
+
+service_postinst ()
+{
+    sync_certificates
+}
+
+service_postupgrade ()
+{
+    sync_certificates
```
Comment (Safihre) [inline spk/mono/src/service-setup.sh]: Why twice? Since it's already done during `postinst`?

Comment (ymartin59) [reply spk/mono/src/service-setup.sh]: Right. I just copied it from previous installer.sh

**Comment by ymartin59** [comment]
@Safihre May you build  this fix and submit as test packages, with hope to receive some feedbacks before I merge and publish in repository ? Thank you in advance

**Comment by Safihre** [comment]
[Building](https://travis-ci.org/Safihre/spksrc/builds/343709692) and updated the link in #3138.

**Comment by ymartin59** [comment]
@Safihre Have you received any feedback about this latest mono package version ?

**Comment by Safihre** [comment]
Not really, there seem to be 38 downloads:
http://www.safihre.com/sabnzbd/stats.html?repo=safihre/spksrc

**Comment by Safihre** [comment]
Tried to test it on my Xpenology but doing an update caused it to not start anymore (makes sense of course 🤒😂).

**Comment by Safihre** [comment]
(DSM update I mean)

**Pull request merged**